src/components/SelectionList/RadioListItem.tsx

```diff
@@ -41,6 +41,7 @@ function RadioListItem<TItem extends ListItem>({
             pendingAction={item.pendingAction}
         >
             <>
+                {!!item.leftElement && item.leftElement}
```

Comment:
It would be amazing to make these changes in a separate PR with clear test steps. Its always tricky to make changes to the low-level components

Reply:
but I assume this should be fine

Reply:
yes, as it's an optional property this shouldn't cause any problem, I also haven't noticed anything off with this component so far